src/stdlib/CreateObject.ts

```diff
@@ -16,6 +16,21 @@ export const CreateObject = new Callable("CreateObject", {
         returns: ValueKind.Dynamic,
     },
     impl: (interpreter: Interpreter, objName: BrsString, ...additionalArgs: BrsType[]) => {
+        let objToMock = objName.value.toLowerCase();
+
+        if (
+            objToMock === "rosgnode" &&
+            additionalArgs[0] &&
```

Comment:
Why only the first  element in `additionalArgs` is checked? Don't you  need to check the rest of the elements?

Reply:
When you call `createObject("roSGNode", arg0)` the first element in additional args is the type of node to create which is what we care about mocking

Reply:
oh I see, makes sense then